src/test/kotlin/no/nav/familie/ef/sak/tilbakekreving/TilbakekrevingServiceTest.kt

```diff
@@ -89,5 +128,47 @@ internal class TilbakekrevingServiceTest {
         assertThat(tilbakekrevingSlot.captured.begrunnelse).isEqualTo(forventetBegrunnelse)
     }
 
+
+    @Test
+    internal fun `Varselbrev må lages med riktig varseltekst`() {
+        val (behandlingId, requestSlot) = mockHentDataForGenereringAvVarselbrev()
```

Comment:
her tenker jeg behandlingId kan være en field /const i klassen, for u unngå att `mockHentDataForGenereringAvVarselbrev` skal returnere Pair, som man må bruke destruct på